### ISSUE TITLE
Bandstructure from wannier90

### ISSUE THREAD

**Issue by poojabasera** [SMTG-UCL/sumo#93]
I have calculated the bandstructure from wannier 90. So i don't have vasprun.xml which has been calculated along a high symmetry path. How sumo-bandstats works in that case in order to calculate the effective masses.

**Comment by ajjackson** [comment]
Wannier90 writes band structures to its own .dat file format, right? We don't support this format for bandplot or bandstats. It might read your vasprun.xml and give some nonsense result based on walking around the SCF k-points as though they are a smooth path.

**Comment by poojabasera** [comment]
Yes sir, it generates its own dat file. Thankyou for your reply.

**Comment by utf** [comment]
I think this could be achieved by easily by writing a wannier to pymatgen band structure parser. Ideally this would be implemented in pymatgen itself. I'll ask around to see if anyone already has something to do this.

Bandplot and bandstats could then be adjusted to accept band structure objects (which I've been meaning to add for a while).

**Comment by ajjackson** [comment]
That seems a sensible approach. In a way phonon-bandplot has taken a step in this direction with the ability to dump/load pymatgen objects via JSON.
